diagnostic.md

```diff
@@ -27,7 +27,16 @@ on the body element. Use that variable to increase font-size of `h1`
 elements by 30%.
 
 ```scss
-// your response here
+
+body {
+  font-size: $base-font-size;
+}
+
+h1 {
+  font-size: $h1-font-size;
+}
+
+$h1-font-size: ($base-font-size * 1.3);
```

Comment:
This idea works, but you'd want to define `$h1-font-size: ($base-font-size * 1.3);` before it's used for the `h1`, otherwise your browser won't know about the variable before it's defined.